api/Window.json

```diff
@@ -6243,10 +6246,10 @@
               "version_added": "9"
             },
             "opera": {
-              "version_added": true
+              "version_added": "3"
```

Comment:
Is this based on testing Opera 3? It's the only so early version in this PR... If there's any doubt, then I'd prefer "≤12.1".

Reply:
I copied this from `pageYOffset`, and then confirmed the results!